packages/colors/README.md

```diff
@@ -32,12 +32,57 @@ In Sass, you can import the files individual by doing:
 @import '@carbon/colors/scss/colors';
 ```
 
-This will make all the colors and token variables available to you in
-your file. They are named with the following structure: `$ibm-color__swatch-grade`, for example:
+This file automatically includes the `carbon--colors` mixin which initializes
+all the color variables for the IBM Design Language.
+
+These color variables follow the naming convention: `$carbon--<swatch>-<grade>`.
+For example:
+
+```scss
+$carbon--blue-50;
+$carbon--cool-gray-10;
+$carbon--black-100;
+$carbon--white-0;
+```
+
+You can also use the shorthand form of these colors by dropping the `carbon--`
+namespace:
+
+```scss
+$blue-50;
+$cool-gray-10;
+$black-100;
+$white-0;
+```
+
+_Note: the shorthand variables require that you do not have any other
+conflicting variables in your setup. Namespaced variables are always preferred
+for this reason, unless you are confident that no collisions will occur._
+
+If you would like you choose when these variables are defined, then you can call
+the `carbon--colors` mixin directly by importing the following file:
+
+```scss
+@import '@carbon/colors/scss/mixins';
+
+// ...
+@include carbon--colors();
+```
+
+#### Migrating from previous versions
+
+If you were originally using a project that had color variables defined as
+`$ibm-color__<swatch>-<grade>`, or are relying on `$ibm-color-map`, you can also
```

Comment:
We use `$ibm-color-map` to extract color + grade with `map-get` and then manipulate the grade by steps in scss. I was wondering if you were considering still keeping the color map in some form? (It's not a hard requirement for us as I think we can work around it, but would mean we don't need to change some of our scss functions 😅 )